R/Data-methods.R

```diff
@@ -560,3 +560,50 @@ setMethod(
     eff
   }
 )
+
+# ngrid ----
+
+## generic ----
+
+#' Number of Doses in Grid
+#'
+#' @description `r lifecycle::badge("stable")`
+#'
+#' A function that gets the number of doses in grid. User can choose whether
+#' the placebo dose (if any) should be counted or not.
+#'
+#' @param object (`Data`)\cr object with dose grid.
+#' @param ... further arguments passed to class-specific methods.
+#' @return `integer` the number of doses in grid.
+#' @export
+#'
+setGeneric(
+  name = "ngrid",
+  def = function(object, ...) {
+    standardGeneric("ngrid")
+  },
+  valueClass = "integer"
+)
+
+## Data ----
+
+#' @rdname ngrid
+#'
+#' @param ignore_placebo (`flag`)\cr should placebo dose (if any) not be counted?
+#'
+#' @aliases ngrid-Data
+#' @example examples/Data-method-ngrid.R
+#'
+setMethod(
+  f = "ngrid",
+  signature = signature(object = "Data"),
+  definition = function(object, ignore_placebo = TRUE) {
```

Comment:
if you have ... for the generic, I think you might need it here too (even though it is unused)